third_party/move/move-compiler-v2/tests/reference-safety/v1-tests/borrow_field_combo_invalid.exp

```diff
@@ -1,42 +1,6 @@
 
 Diagnostics:
-error: cannot copy mutable reference in local `inner`
-   ┌─ tests/reference-safety/v1-tests/borrow_field_combo_invalid.move:35:30
-   │
-35 │         let c; if (cond) c = copy inner else c = &mut outer.s1;
-   │                              ^^^^^^^^^^ copied here
-
-error: cannot copy mutable reference in local `inner`
-   ┌─ tests/reference-safety/v1-tests/borrow_field_combo_invalid.move:13:30
-   │
-13 │         let c; if (cond) c = copy inner else c = &mut outer.s1;
-   │                              ^^^^^^^^^^ copied here
-
-error: cannot copy mutable reference in local `inner`
-   ┌─ tests/reference-safety/v1-tests/borrow_field_combo_invalid.move:46:37
-   │
-46 │         let c; if (cond) c = id_mut(copy inner) else c = &mut outer.s1;
-   │                                     ^^^^^^^^^^ copied here
-
-error: cannot copy mutable reference in local `inner`
-   ┌─ tests/reference-safety/v1-tests/borrow_field_combo_invalid.move:24:37
-   │
-24 │         let c; if (cond) c = id_mut(copy inner) else c = &mut outer.s1;
-   │                                     ^^^^^^^^^^ copied here
-
-
-Diagnostics:
-error: cannot copy mutable reference in local `inner` which is still borrowed
-   ┌─ tests/reference-safety/v1-tests/borrow_field_combo_invalid.move:58:18
-   │
-57 │         let c; if (cond) c = &mut inner.f1 else c = &mut inner.f2;
-   │                              -------------          ------------- previous mutable field borrow
-   │                              │
-   │                              previous mutable field borrow
-58 │         let f1 = &inner.f1;
-   │                  ^^^^^^^^^ copied here
-
-error: cannot immutable borrow local since other mutable references exist
+error: cannot immutable borrow local `inner` since other mutable references exist
```

Comment:
Please fix this message, so we don't have to file a bug for it.  The source is not quite in scope for this review, so can't make this comment there.

`reference_safety_processor.rs:624`: "cannot mutable borrow" --> "cannot mutably borrow"

`reference_safety_processor.rs:637`: "cannot immutable borrow" --> "cannot immutably borrow"